.github/workflows/store-artefact.yml

```diff
@@ -25,7 +25,10 @@ jobs:
         with_examples: ["True"]
         package_option: ["-DWITH_ALL_PACKAGES=ON", "-DWITH_STABLE_PACKAGES=ON"]
         language_bindings:
-          ["-DWITH_JAVA=True -DWITH_CSHARP=True -DWITH_PYTHON=True"]
+          [
+            "-DWITH_JAVA=True -DWITH_CSHARP=True -DWITH_PYTHON=True
+            -DWITH_R=True",
```

Comment:
does the newline within the string cause an issue?

Reply:
This is the VS Studio Code autoformatting tool's fault. I am planning a formatting-only PR after this, but this slipped in now somehow.

It doesn't cause an issue and translates correctly in the run linked in the PR description to 
![image](https://user-images.githubusercontent.com/10500965/124149719-c75b7800-da88-11eb-8f47-21ddc828c3a2.png)